recipes/diffpy.labpdfproc/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{%- set name = "diffpy.labpdfproc" -%}
+{%- set version = "0.1.2" -%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/diffpy_labpdfproc/diffpy_labpdfproc-{{ version }}.tar.gz
+  sha256: 521dd9a24c062ca52d69ef5e571c8551dbff04a23bad55bb8a7cf5940eba9bf6
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Could you change this to `script: {{ PYTHON }} ...`?

Reply:
Fixed

```
script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```